matplotcheck/tests/test_notebook.py

```
@@ -0,0 +1,145 @@
+        "_ih": ["", 'print("Bad notebook")', "import os", "locals()"],
+        "In": ["", 'print("Bad notebook")', "import os", "locals()"],
+        "Out": {},
+        "_i": "import os",
+        "_ii": 'print("Bad notebook")',
+        "_iii": "",
+        "_i1": 'print("Bad notebook")',
+        "_i2": "import os",
+        "_i3": "locals()",
+    }
+
+
+def test_notebook_convert_single_axes(basic_polygon_gdf):
+    """Test convert single axes."""
+    fig, ax = plt.subplots()
+    basic_polygon_gdf.plot(ax=ax)
+    store = nb.convert_axes(plt)
+    with pytest.raises(AssertionError, match="Axis lines are displayed on"):
+        PlotTester(store).assert_axis_off()
+    plt.close()
+
+
+def test_notebook_convert_multi_axes(basic_polygon_gdf):
+    """Test convert multi axes."""
+    fig, (ax1, ax2) = plt.subplots(1, 2)
+    basic_polygon_gdf.plot(ax=ax1)
+    basic_polygon_gdf.plot(ax=ax2)
+    store = nb.convert_axes(plt, which_axes="all")
+    for axes in store:
+        with pytest.raises(
+            AssertionError, match="Axis lines are displayed on"
+        ):
+            PlotTester(axes).assert_axis_off()
+    plt.close()
+
+
+def test_notebook_convert_last_axes(basic_polygon_gdf):
+    """Test convert last axes."""
+    fig, (ax1, ax2) = plt.subplots(1, 2)
+    basic_polygon_gdf.plot(ax=ax2)
+    store = nb.convert_axes(plt, which_axes="last")
+    with pytest.raises(AssertionError, match="Axis lines are displayed on"):
+        PlotTester(store).assert_axis_off()
+    plt.close()
+
+
+def test_notebook_convert_first_axes(basic_polygon_gdf):
+    """Test convert first axes."""
+    fig, (ax1, ax2) = plt.subplots(1, 2)
+    basic_polygon_gdf.plot(ax=ax1)
+    store = nb.convert_axes(plt, which_axes="first")
+    with pytest.raises(AssertionError, match="Axis lines are displayed on"):
+        PlotTester(store).assert_axis_off()
+    plt.close()
+
+
+def test_notebook_convert_axes_error(basic_polygon_gdf):
+    """Test convert axes error."""
```

Incorrect parameter value for which_axes should return a value error.